backend/src/users/users.module.ts

```diff
@@ -1,17 +1,24 @@
 import { Module } from '@nestjs/common';
 import { MongooseModule } from '@nestjs/mongoose';
-import { UsersService } from './users.service';
+import { JwtModule } from '@nestjs/jwt';
 import { UsersController } from './users.controller';
+import { UsersService } from './users.service';
 import { User, UserSchema } from './users.schema';
-import { RolesModule } from '../roles/roles.module';
+import { Role, RoleSchema } from '../roles/roles.schema';
 
 @Module({
-  imports: [
-    RolesModule,
-    MongooseModule.forFeature([{ name: User.name, schema: UserSchema }]),
-  ],
-  providers: [UsersService],
-  controllers: [UsersController],
-  exports: [UsersService]
+    imports: [
+        MongooseModule.forFeature([
+            { name: User.name, schema: UserSchema },
+            { name: Role.name, schema: RoleSchema }
+        ]),
+        JwtModule.register({
```

Comment:
There is already a JWTModule registered in the AuthModule. Maybe use the AuthModule ?